code.js

```diff
@@ -0,0 +1,10 @@
+function MobileMenu () {
+  if (document.getElementById('MobileMenu').style.display == 'block') {
+    document.getElementById('MobileMenu').style.display = 'none';
+  } else {
+     document.getElementById('MobileMenu').style.display = 'block';
+    }
+}
+  
+document.getElementById ('menu-icon').addEventListener('click', MobileMenu);
+document.getElementById ('close-menu').addEventListener('click', MobileMenu);
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you first assign data of `document.getElementById ('menu-icon')`, `document.getElementById ('close-menu')`, and `document.getElementById('MobileMenu')` to thesir specific **variables**  and then you can use those **variables** instead of writing every time `document.getElementById('...')` for JavaScript best practices.